### PR TITLE
LESQ-278 Feat/home page nav

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

- Add homePageNav to storybook
- fully functional and style

## Issue(s)

Fixes #

## How to test

1. Go to

https://deploy-preview-1898--owa-storybook.netlify.thenational.academy/?path=/story/navigation-homepagetabimagenav--home-page-tab-image-nav

2. Click on Navigation/HomePageNav
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/components/HomePageTabImageNav/HomePageTabImageNav.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+// export default meta;
+// type Story = StoryObj<typeof Component>;
+
+// export const HomePageTabImageNav: Story = {
+//   args: {},
+//   render: (args) => {
+//     const [current, setCurrent] = useState("teachers");
+//     return <Component {...args} current={current} setCurrent={setCurrent} />;
+//   },
+// };
```
Comment (njblore) [inline src/components/HomePageTabImageNav/HomePageTabImageNav.stories.tsx:21]: can we remove this?

File: src/components/HomePageTabImageNav/HomePageTabImageNav.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { Dispatch, SetStateAction } from "react";
+
+import { FlexProps } from "../Flex";
+import Flex from "../Flex/Flex";
+import HomePageTabImageButton from "../NewButton.tsx/HomePageTabImageButton";
+import { Hr } from "../Typography";
```
Comment (njblore) [inline src/components/HomePageTabImageNav/HomePageTabImageNav.tsx:6]: I think we're aiming to use aliased imports everywhere instead of relative ones

**Review by njblore — APPROVED** [review]

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
- For some reason they have doubled up names for assistive tech (see in the green box, a very useful tab in chrome!)
Screenshot 2023-09-13 at 09 43 51
- When it moves to mobile width, the "new" should be removed from the curric maps
Screenshot 2023-09-13 at 09 44 20
- Should these all be within a nav? Or at least within a list to show they are connected

**Review by Monika-Koziol — CHANGES_REQUESTED** [review]
A couple of things to check:

- [ ] after clicking on a tab the line should be black rather then yellow and black (that's a focus state for tabbing with a keyboard)
- [x] there is a small white gap between the black underline and green background
- [x] it breaks on small mobile: hide new tag, can we change font size to 14px?

**Comment by jack-skerrett-oak** [comment]
> 

Not to say it should or shouldn't, but currently across the site, focus remains on an element until a user clicks away. Not sure if there is anything we can do, that is just the current state of things!

**Review by Monika-Koziol — COMMENTED** [review]
Screenshot 2023-09-13 at 13 41 46
It still looks broken on a mobile. Am I looking at the right thing?

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
Two of my initial requests still need addressing:
- New tag doesn't disappear at narrow widths (as well as the issue Monika has shown where the icons bleed off the edge of the page, I assume everything should just get smaller once there is no space?)
- They are still just three buttons in a div. They should either be in a nav/list, or just a list. Not sure which is more applicable here.

**Review by jack-skerrett-oak — APPROVED** [review]

**Review by Monika-Koziol — APPROVED** [review]
Looking good.

**Pull request merged**